### PR TITLE
Add rebuild API endpoint

### PR DESCRIPTION
**Pull request by KrauseFx**
Related to https://github.com/fastlane/ci/issues/451

### CODE REVIEW

File: app/features-json/build_json_controller.rb [hunk 1]
```diff
@@ -13,6 +13,70 @@ class BuildJSONController < APIController
...
+        work_queue: FastlaneCI::GitRepo.git_action_queue, # using the git repo queue because of https://github.com/ruby-git/ruby-git/issues/355
+        trigger: project.find_triggers_of_type(trigger_type: :manual).first,
+        git_fork_config: git_fork_config
+      )
+      build_runner.setup(parameters: nil)
+      Services.build_runner_service.add_build_runner(build_runner: build_runner)
+
+      json({
+        status: :success,
+        build_number: build_runner.current_build.number
```
Comment (nakhbari) [inline app/features-json/build_json_controller.rb]: Could we return the build summary? That way we can add the new build to the build table in the project details page

Comment (KrauseFx) [reply app/features-json/build_json_controller.rb]: Ah yeah, great idea 👍 Will update

**Comment by KrauseFx** [comment]
Alright, ready to be merged from my side @snatchev. I'd love to add tests on this one also, however let's get this merged first. Do we already have some good tests for authenticated API endpoints I can take a look at on how to stub the auth flow?

**Comment by snatchev** [comment]
@KrauseFx We have some examples for specs in authenticated API controllers here: https://github.com/fastlane/ci/blob/master/spec/features-json/api_controller_spec.rb#L45

All you have to do is set the header. The bigger challenge, IMO is going to be stub `current_project` similar data models.

**Comment by vicdes77** [comment]
I'm am not subscribed to any of these codes someone is doing this to my phone I want it reversed. Someone is getting and using all my info through doing whatever these things are. I need help fixing it

**Pull request merged**